core/utils/config_utils.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+
+	"log"
+
+	"gopkg.in/yaml.v3"
+
+	"os"
+	"path/filepath"
+)
+
+const (
+	PngMimeType = "image/png"
+
+	TextRegex = `^[a-zA-Z0-9 +.,;:?!'’"“”\-_/()\[\]~&#$—%]+$`
+
+	TextCharsLimit = 500
+)
+
+var (
+	// ImageExtensions List of common image file extensions
+	// Only support PNG for now to reduce surface area of image validation
+	// We do NOT want to support formats like SVG since they can be used for javascript injection
+	// If we get pushback on only supporting png, we can support jpg, jpeg, gif, etc. later
+	ImageExtensions = []string{".png"}
+)
```

Comment:
Do we need this?

Reply:
v1.1.12 included utility functions for parsing JSON and YAML files which we used. There were eliminated in v1.1.13, I reimplemented them within our library.

Reply:
Yes but I mean the constants and ImageExtensions.

Reply:
I don't think we need the image part of the code

Reply:
I agree, the consts and the var should go.